diagnostic.md

```diff
@@ -150,7 +183,10 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+|2              1|
+|3               |
+|4               |
+|5               |
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).